plugins/core/IO/qLASIO/src/LasOpenDialog.cpp

```diff
@@ -31,13 +31,19 @@ static QListWidgetItem* CreateItem(const char* name)
 	return item;
 }
 
-static bool IsCheckedIn(const QString& name, const QListWidget& list)
+static bool IsCheckedIn(const QString& name, const QListWidget* list)
 {
-	for (int i = 0; i < list.count(); ++i)
+	if (!list)
 	{
-		if (list.item(i)->text() == name)
+		assert(false);
+		return false;
+	}
+
+	for (int i = 0; i < list->count(); ++i)
+	{
+		if (list->item(i)->text() == name)
 		{
-			return list.item(i)->checkState() == Qt::Checked;
+			return list->item(i)->checkState() == Qt::Checked;
 		}
 	}
```

Comment:
Not sure its worth changing the input from `const QListWidget& list` to `const QListWidget* list`

Reply:
It was to be able to test the pointer at one place (instead of dereferencing it without testing it, or testing it at 2 different places).